docs/algorithms/afl/implementation_ja.md

```diff
@@ -45,7 +43,13 @@ fuzzufにおいてAFLは `include/fuzzuf/algorithms/afl` および `algorithms/a
     );
 ```
 
-これらのノードの内、`bit_flip1`, `bit_flip_other`などのノードのルーチンは、`include/fuzzuf/algorithms/afl/afl_mutation_hierarflow_routines.hpp` で宣言されています。`normal_update`, `construct_auto_dict`などのノードのルーチンは、`include/fuzzuf/algorithms/afl/afl_update_hierarflow_routines.hpp` で宣言されています。それらのノード以外のルーチンは、 `include/fuzzuf/algorithms/afl/afl_other_hierarflow_routines.hpp` で宣言されています。
+各ノードのルーチンは次のコードで実装されています:
+- `bit_flip1` and `bit_flip_other`:
```

Comment:
I think this wording would bring a confusion (the meaning changes before and after). 
I prefer something like "mutation routines (e.g., `bit_flip1` and `bit_flip_other`)" compared to this.

Reply:
Fixed with [fd84959](https://github.com/fuzzuf/fuzzuf/pull/20/commits/fd8495922ac0eb726699c0f7188aab47e7c7b44c).